### ISSUE TITLE
MS Studio 4.4.x - Malfunction of the special control notes in VST Native Instruments - KONTAKT

### ISSUE THREAD

**Issue by VladaScore** [musescore/MuseScore#24980]
### Issue type

VST bug

### Description with steps to reproduce

1. I open an existing score with a VST Accordion.
2. I bypass current Studio crash (#24956). See below the explanation how-to.
3. I click the Play button or press the space-key.
4. I check and confirm that everything is played fine until the Prima Volta.
5. After the Prima Volta, in the repetition phase the Accordion switches are screwed, and they malfunction.
6.  For example, control note C1 should make accordion to play soprano, but it keeps playing previous sound of clarinet (control note C2) which was the last sound before the Prima Volta. Usually, it gets stuck to the end of session - file closure. Rarely, some of the incoming control notes might switch the sound, but incorrectly (like shifted).

NOTE: 
a) **This never ever malfunction during the very first session with a brand-new music score.** Once a music score gets closed and reloaded malfunction arises.
b) Also, this malfunction has never happened with the same KONTAKT-7.10.1 and Muse Score Studio 4.3.x.

### Supporting files, videos and screenshots

[MalfunctionOfKontaktAccordionSwitches.zip](https://github.com/user-attachments/files/17190811/MalfunctionOfKontaktAccordionSwitches.zip)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Windows 11 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.2-242570931, revision: github-musescore-musescore-3130f97

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Windows 11 Enterprise (23H2), Windows 10 Home (22H2)

### Additional context

**Important information:**
Native Instruments - **KONTAKT uses a specific range of the special control notes not to produce their sound, but as control sequence for a certain action**.
In the case of the VST Accordion, notes/keys from A#0 to B1 are switches that should change the accordion sounds between soprano, clarinet, bassoon, piccolo, organ, ..., and master.
Also, notes/keys from C2 to D2 are for playing styles: gently, normal or sharp.

I have one of the special control notes (usually B0) in the very first takt of all my scores.
Every KONTAKT user has to set it, otherwise no switch chosen no accordion sound produced.

My workaround to bypass Studio crash is as following:
1. I open an existing score.
2. I go to Mixer and change instruments from VST into Muse Basic.
3. I click Play button and let the score be played.
4. I change instrument back to VST Accordion.
5. MS Studio now plays the score, but with the VST malfunction explained above.

Thanks,
Vlada

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by bkunda** [comment]
@RomanPudashkin could you pls take a quick look at this?

**Comment by VladaScore** [comment]
@RomanPudashkin As you might have already noticed from my .mscz sample attached above, for the better readability, I write these "special" notes for the key switches not as a usual Voice#1, but as a Voice#2. It shouldn't matter, but just to be aware of it.

**Comment by VladaScore** [comment]
Yes, it DID work in all versions of MuseScore Studio 4.3.x. I clearly
stated that in the issue description.

Sorry for the potential confusion, but I have not decided myself the type
of regression neither I clearly understand what it exactly means for the
development team :-(

Regards,
Vlada
